### PR TITLE
First draft of Lottie roadmap page

### PR DESCRIPTION
**Pull request by b-wils**
Please leave comments and content and format.

### CODE REVIEW

File: content/templates/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+image: null
+---
+
+### v1.0 (September 2024)
+
+The v1.0 specification contains features that are commonly used by current players and that behave in a consistent way. Not all features are currently covered and missing features may be added in future releases.
+
+Features in 1.0 release:
+
+- **Layers** - Shapes, Solids, Images, Precomposition
```
Comment (mbasaglia) [inline content/templates/roadmap.md:14]: null layers too

File: content/templates/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- **Shapes** - Rectangle, Ellipse, Path, Polystar
+- **Shape Styles and Modifiers** - Fill, Stroke, Gradient Fill, Gradient Stroke, Trim Path
+- **Shape Grouping** - Group, Transform
+- **Transforms** - Position, Split Position, Rotation, Scale, Opacity, Skew, Skew Axis
+- **Assets** - Precomposition, Image
+- **Time Remap and Stretch**
+- **Masks**
```
Comment (mbasaglia) [inline content/templates/roadmap.md:21]: and mattes

File: content/templates/roadmap.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- Contents TBD
+
+### Future Releases
+
+- Player Profiles
+- Text Layers
+- Layer effects
+- Metadata
+- Blend modes
+- Expressions
```
Comment (mbasaglia) [inline content/templates/roadmap.md:35]: Should we mention additional modifiers?

**Review by Aidosmf — APPROVED** [review]
LGTM

**Review by mbasaglia — APPROVED** [review]

**Review by fmalita — APPROVED** [review]

**Pull request merged**